sources/packages/api/src/services/sin-validation/sin-validation.service.ts

```diff
@@ -98,7 +100,7 @@ export class SINValidationService extends RecordDataModelService<SINValidation>
       newSINValidation.sinEditedNote = savedNote;
       newSINValidation.creator = auditUser;
       newSINValidation.createdAt = now;
-      newSINValidation.user = student.user;
+      newSINValidation.student = student;
```

Comment:
` newSINValidation.student = {id:student.id} as student;`

Reply:
as the student instance was already present, used the same instead of creating a new instance. Also there is no cascade from sinvalidation to student so any other property other than id will be disregarded.